try{
    num1 = 6
    num2 = 0

    console.log(num1/num2);
}
catch(err)
{
    console.log(err)
}
